.travis.yml

```diff
@@ -159,15 +159,10 @@ jobs:
 
     - stage: whitesource
       script: git branch -f "$TRAVIS_BRANCH" && git checkout "$TRAVIS_BRANCH" && cp .jvmopts-travis .jvmopts && sbt whitesourceCheckPolicies whitesourceUpdate
-    
+
     - stage: publish
-      script: cp .jvmopts-travis .jvmopts && sbt ++2.12.11 publish && git status && sbt ++2.13.3 akka-grpc-runtime/publish ++2.13.3! akka-grpc-codegen/publish
-      name: sbt publish
-    - script: openssl aes-256-cbc -K $encrypted_9c9c33071881_key -iv $encrypted_9c9c33071881_iv -in gradle.properties.enc -out ./gradle-plugin/gradle.properties -d && cd gradle-plugin && ./gradlew publishPlugins
+      script: openssl aes-256-cbc -K $encrypted_9c9c33071881_key -iv $encrypted_9c9c33071881_iv -in gradle.properties.enc -out ./gradle-plugin/gradle.properties -d && cd gradle-plugin && ./gradlew publishPlugins
```

Comment:
This looks like a left-over. 😅

Reply:
no, I didn't convert the gradle publishing yet (because it needs that travis-encrypted `gradle.properties` file)

Reply:
I see, this is still DRAFT. I'll hold.